### ISSUE TITLE
Empty enumerables aren't sent as people properties to MixPanel 

### ISSUE THREAD

**Issue by yaronguez** [eealeivan/mixpanel-csharp#3]
If we send an empty list as a property in the PeopleSet method, it isn't sent to MixPanel. I'm trying to empty an existing list. How is the best way to handle this?

Thanks!

**Comment by yaronguez** [comment]
Follow up: I found that sending an IEnumerable in the following manner allows an empty list to be sent along to Mixpanel:
```
mixPanelClient.PeopleSet(distinctId, new
{
    SomeList = someList.DefaultIfEmpty(string.Empty),
});
```

**Comment by eealeivan** [comment]
Hi, I added a fix https://github.com/eealeivan/mixpanel-csharp/commit/b564ae9a0f8fd630351f22336796075a1b03d45e that will send empty enumerables too. I want to fix few more issues and then release new minor version.

**Comment by eealeivan** [comment]
This bug is fixed in version 4.0.0.
